README.md

```diff
@@ -1,151 +1,20 @@
-# News Summary challenge
```

Comment:
Opened App, 
- It worked on the browser with no tweaks
- All tests passing
- Looks great, simple and workable interface. 
- Liked the testing suite - clearly readable
- I like how you stored the API call in local storage, reduced calls to the API, but how would it work if you want new information? This also avoids the need to mock the test if you run the website first.
- Buttons at top don't do anything - assuming this is a future feature.